config/rabe-fs-backup.timer

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Run RaBe Backup Service timer daily
+
+[Timer]
+# Run once per day
+OnCalendar=daily
+# Start immediately if last start was missed (ie when machine was down)
+Persistent=true
+# Delay timer randomly to spread out load (centos-7.3 and up only)
+RandomizedDelaySec=30min
+# Do not coalesce with other timers, no need to save power during a backup
+AccuracySec=1us
```

Comment:
`AccuracySec` is about at which accuracy a timer shall fire, which would normally be in a 1 minute window from the `OnCalendar` value.  Again, if you randomize it's start, there's really no point in having this option...